luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/ArithmeticMakerFactory.java

```diff
@@ -21,14 +21,14 @@
  */
 public class ArithmeticMakerFactory implements Factory<MetricMaker> {
 
-    public static final String OPERATION = "operation";
+    protected static final String FUNCTION = "function";
```

Comment:
Why the change here from `operation` to `function`? I picked `operation` originally because that's what addition, subtraction, multiplication and division are called. People don't usually call them functions, even if that's what they technically are.

Reply:
sadly, it's called function in the ArithmeticMaker.  I gave my blessing to aligning with that, even though I wish operation was what we had chosen.

Reply:
People writing config don't care what it's called in the implementation of ArithmeticMaker. With Luthier, people are probably almost never going to look at the ArithmeticMaker.

Reply:
ok.  you've changed my mind.  call it operation until the last minute.

Reply:
Although that is true, I believe aligning with "function" makes it simpler to track down issues from our side. Although I also find it some what awkward, people coming from Druid would most likely expect this idiom, as you can see here:
https://druid.apache.org/docs/latest/querying/post-aggregations.html#javascript-post-aggregator
The name "function" is a middle ground that I believe is kind of reasonable in this case.

Reply:
I don't see how this makes it easier for us to track down issues.

Reply:
I will be using the "operation" in place of the current "function"